### ISSUE TITLE
license agreement not stored, asked multiple times

### ISSUE THREAD

**Issue by hohwille** [devonfw/ide#275]
```
Do you accept these terms of use and all license agreements? (yes/no) yes
/c/projects/latest/scripts/command/../functions: line 212: /.devon/.license.agreement: No such file or directory
```

Seems that `DEVON_HOME_DIR` is not set at this state hence resulting in `/.devon/.license.agreement`.

**Comment by hohwille** [comment]
Seems to be a bug due to my testing when an old version of `devon` script was left from an old installation I tried ti update.
Now I however got this output:
```
Latest version seems unstable: 3.0.0-beta23\n Are you sure you want to install this version? (yes/no)
```
So it seems to miss an `echo -n` here.

**Comment by hohwille** [comment]
Initial problem was caused from an old `devon-ide` release and has already been fixed long ago (#115).
Second problem fixed with PR #278
